.github/workflows/test-and-deploy.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - name: Create GitHub Release
         uses: sendgrid/dx-automator/actions/release@main
         with:
+          assets: sendgrid-java.jar
```

Comment:
Should replace the `mvn install` with `make install` or the jar won't get copied to the root. Same for other java repos.